psydac/ddm/cart.py

```diff
@@ -226,6 +222,13 @@ def shape( self ):
     def subcomm( self ):
         return self._subcomm
 
+    #---------------------------------------------------------------------------
+    def topetsc( self ):
```

Comment:
I know that this file does not contain many docstrings... Nevertheless, can you please add one to this method at least?